spec/requests/api/v1/fees_spec.rb

```diff
@@ -73,4 +73,45 @@
       end
     end
   end
+
+  describe 'PUT /fees/:id' do
+    let(:customer) { create(:customer, organization:) }
+    let(:subscription) { create(:subscription, customer:) }
+    let(:fee) { create(:charge_fee, fee_type: 'instant_charge', subscription:, invoice: nil) }
+
+    let(:update_params) { { payment_status: 'succeeded' } }
+
+    it 'updates the fee' do
+      put_with_token(organization, "/api/v1/fees/#{fee.id}", fee: update_params)
+
+      aggregate_failures do
+        expect(response).to have_http_status(:success)
+
+        expect(json[:fee][:lago_id]).to eq(fee.reload.id)
+        expect(json[:fee][:lago_group_id]).to eq(fee.group_id)
+        expect(json[:fee][:item][:type]).to eq(fee.fee_type)
+        expect(json[:fee][:item][:code]).to eq(fee.item_code)
+        expect(json[:fee][:item][:name]).to eq(fee.item_name)
+        expect(json[:fee][:amount_cents]).to eq(fee.amount_cents)
+        expect(json[:fee][:amount_currency]).to eq(fee.amount_currency)
+        expect(json[:fee][:vat_amount_cents]).to eq(fee.vat_amount_cents)
+        expect(json[:fee][:vat_amount_currency]).to eq(fee.vat_amount_currency)
+        expect(json[:fee][:units]).to eq(fee.units.to_s)
+        expect(json[:fee][:events_count]).to eq(fee.events_count)
+        expect(json[:fee][:payment_status]).to eq(fee.payment_status)
+        expect(json[:fee][:created_at]).to eq(fee.created_at&.iso8601)
+        expect(json[:fee][:succeeded_at]).to eq(fee.succeeded_at&.iso8601)
+        expect(json[:fee][:failed_at]).to eq(fee.failed_at&.iso8601)
+        expect(json[:fee][:refunded_at]).to eq(fee.refunded_at&.iso8601)
```

Comment:
is that possible to have only one expectation here? 😅  something like `have_attributes(...)` but for your json variable